src/components/Cluster/ClusterDetail/ClusterDetailView.js

```diff
@@ -292,7 +292,7 @@ class ClusterDetailView extends React.Component {
 
     if (!cluster) return null;
     const { id, owner, release_version } = cluster;
-    const release = releases[release_version] ?? null;
+    const release = releases[release_version];
```

Comment:
The logic in other components already consider that this can be `undefined`, and not `null`.